app/cdap/components/PipelineDetails/PipelineDetailsTopPanel/PipelineDetailsButtons/PipelineHistoryButton.tsx

```diff
@@ -15,12 +15,16 @@
  */
 
 import { Button } from '@material-ui/core';
-import React, { useRef, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import styled from 'styled-components';
 import classnames from 'classnames';
-import { PipelineHistory } from 'components/PipelineHistory/PipelineHistory';
+import PipelineHistoryOuter from 'components/PipelineHistory/PipelineHistory';
```

Comment:
Did we confirm from UX that this button is shown as just 'history' ? I think there was some discussion around the naming

Reply:
This was based on the figma design

Reply:
yeah I think we should rename it - lets have a discussion with UX @de-lan